### DISCUSSION TITLE
Radiolib with atmega644

### DISCUSSION THREAD

**Discussion by jontxu70** [jgromes/RadioLib#906] [General]
I have my own atmega644pa board.

I want to develop and test lora wireless communication

I've been testing with no luck with default values.

https://github.com/RoelKroes/

I am not sure pin mapping is ok.

// atmega644 // atmega pro mini
#define SCKpin 7 // SCK 13
#define SSpin 4 // SS 10
#define MOSIpin 5 // MOSI 11
#define MISOpin 6 // MISO 12

Is there any way to change pin mapping?

Can anyone help me?

Best regards

**Comment by jgromes** [comment]
> Is there any way to change pin mapping?

That depends on your chosen MCU - does ATmega644 allow changing the SPI pins? Other ATmega MCUs (like ATmega328) do not allow this, the SPI pins are fixed.

I suggest asking in generic Arduino forums about this, I can only help you with problesm specific to this library.
